### ISSUE TITLE
Refactoring

### ISSUE THREAD

**Issue by BladeMF** [phpactor/vscode-phpactor#8]
I am not sure if I will have the time, but still, somebody might if I can't, so...

### How can we enable all the refactorings for VS Code? 
I mean in terms of calling Phpactor (I can research the corresponding code for VS Code). I noticed some of them are available on the command line while others seem to be more tricky to invoke and are available only in VIM. 

Considering how few tools do refactoring for PHP (I found only this one), even slow implementation should be better than nothing.

**Comment by dantleech** [comment]
This is more a question of enabling them for the language server, and probably better raised in https://github.com/phpactor/phpactor where other people can see it.

In general this is mostly just a question of effort, but it is ultimately the goal to have all the refactorings available from the language server.

**Comment by dantleech** [comment]
Currently you should have a few code actions (import missing classes, add missing properties, complete constructor etc)

**Comment by BladeMF** [comment]
Feel free to move this issue. Or should I create another one?

If it is about changing the language server I am probably out of my depth there.

**Comment by BladeMF** [comment]
> Currently you should have a few code actions (import missing classes, add missing properties, complete constructor etc)

Yes, I saw one of them - the Import. Not sure how to test the others.
